src/components/inputs/Input/Mark.tsx

```diff
@@ -59,7 +61,9 @@ export function Mark({
           id={id}
           name={name}
           onChange={onChange}
+          onClick={onClick}
```

Comment:
Hey @hedyyytang ! If I'm not mistaken, there's very little difference between `onClick` and `onChange` on most browsers, and I was just curious about why you need to add an `onClick` here.

Reply:
Good point. I removed it. It was originally added as an option since the [design doc on Radio](https://vimeo.github.io/iris/sb/main/?path=/docs/components-radio--controls) has an onClick option.